src/Page/Textile/Simulator.elm

```diff
@@ -506,6 +538,13 @@ commandsForNoModal modal =
                     |> Task.attempt (always NoOp)
                 ]
 
+        SelectProductModal _ ->
+            Cmd.batch
+                [ Ports.removeBodyClass "prevent-scrolling"
+                , Dom.focus "selector-product"
+                    |> Task.attempt (always NoOp)
```

Comment:
This doesn't seem to set the scroll position at the right height, the jump making hard to understand why it's happening and losing context

Reply:
Sure thing, here's a screecast

Uploading REC-20240122144003.mp4…